### DISCUSSION TITLE
Selected region

### DISCUSSION THREAD

**Discussion by moranmono** [MadeiraAlexandre/Cronica#36] [Feedback]
Hi. 
First of all, this is a great app—more robust and fluid than anything else I looked for in the app stores, both Apple and Android.

What region should I select if mine isn't on the list?
How do you determine the region? 

When I choose the wrong region, I get a lot of titles that aren't available to me.
Thanks.

**Comment by kevinm6** [comment]
> What region should I select if mine isn't on the list?
> 
> How do you determine the region? 

I'm not the Dev of the App but I think that the regions available are the ones that [TMDb API](https://developer.themoviedb.org/reference/watch-providers-available-regions) offers, and you can find them [here](https://github.com/egger/cronica/blob/main/Shared/View/WatchProviders/WatchProvidersList.swift#L236) and [here](https://github.com/egger/cronica/blob/main/Shared/Model/WatchProviders.swift#L55)

**Reply by moranmono** [reply]
Thanks. Do you think it's possible to add a region to the list? Should I open an issue?

**Reply by kevinm6** [reply]
Maybe you should send a **feedback** to **TMDb support** and ask them to support your region.
But the app's dev @MadeiraAlexandre might had a better idea.

**Reply by MadeiraAlexandre** [reply]
Hi, I think that this is more up to JustWatch support than TMDB itself. If your region is supported by JustWatch, I can open a question on the support forum about it.
It could also be the case for outdated list of region in the app itself since it is hardcoded, what is your region code? I can check it out.

**Reply by moranmono** [reply]
Thank. My region is "he-IL". I couldn't find it in the files @kevinm6 linked to in his response.

**Reply by MadeiraAlexandre** [reply]
https://github.com/egger/cronica/assets/33880764/7cf06566-b01f-4f88-bc1d-820a47730efc

They do support and I've updated the code to reflect the change, thank you for the report! 
Also, can you please confirm the streaming platforms that appears in the video?

**Reply by moranmono** [reply]
Thanks a lot. I think so, although I must say that I am not familiar with most of the names on the list. I will check it and report back.

**Reply by MadeiraAlexandre** [reply]
There's a new update on the App Store with this change, please give it a try when you can.

**Reply by moranmono** [reply]
Thank you. I'm checking it now.

**Reply by moranmono** [reply]
Thanks for the update. It's working great.
